src/app/pages/OnDemandRadioPage/index.test.jsx

```diff
@@ -11,19 +11,20 @@ import { RequestContextProvider } from '#contexts/RequestContext';
 import OnDemandRadioPage from '.';
 import pashtoPageData from '#data/pashto/bbc_pashto_radio/w3ct0lz1';
 import koreanPageData from '#data/korean/bbc_korean_radio/w3ct0kn5';
+import zhongwenPageData from '#data/zhongwen/bbc_cantonese_radio/w172xf3r5x8hw4v';
 import indonesiaPageData from '#data/indonesia/bbc_indonesian_radio/w172xh267fpn19l';
 import afaanoromooPageData from '#data/afaanoromoo/bbc_afaanoromoo_radio/w13xttnw';
 import * as analyticsUtils from '#lib/analyticsUtils';
 import { ToggleContextProvider } from '#contexts/ToggleContext';
 import getInitialData from '#app/routes/onDemandRadio/getInitialData';
 
-const Page = ({ pageData, service, isAmp = false }) => (
+const Page = ({ pageData, service, isAmp = false, variant }) => (
```

Comment:
Does variant need a default value (is it `default` or undefined)?

Reply:
It would be `undefined` here for the tests where I'm not passing it in, and it defaults to `default` in the `ServiceContextProvider` component.

Would assigning it a default value of 'default' be the correct approach [here (ln 21)?](https://github.com/bbc/simorgh/pull/7174/files#diff-c2ab92d1dd48695dee1007df0a1452aaR21) Or is leaving it as undefined for the other tests OK?

Reply:
Maybe just leave it 🤷 ?

Reply:
Aye agreed.